sycl/test/check_device_code/cuda/ldg.cpp

```diff
@@ -9,279 +10,146 @@ using namespace sycl;
+// CHECK-OPAQUE: tail call i64 @llvm.nvvm.ldg.global.i.i64.p0(ptr nonnull %{{.*}}, i32 8)
+template SYCL_EXTERNAL sycl::vec<unsigned long, 3> sycl::ext::oneapi::experimental::cuda::ldg(const sycl::vec<unsigned long, 3> *);
+// CHECK-OPAQUE: tail call <2 x i64> @llvm.nvvm.ldg.global.i.v2i64.p0(ptr %{{.*}}, i32 16)
+template SYCL_EXTERNAL sycl::vec<unsigned long long, 2> sycl::ext::oneapi::experimental::cuda::ldg(const sycl::vec<unsigned long long, 2> *);
+// CHECK-OPAQUE: tail call <2 x i64> @llvm.nvvm.ldg.global.i.v2i64.p0(ptr %{{.*}}, i32 16)
+// CHECK-OPAQUE: tail call i64 @llvm.nvvm.ldg.global.i.i64.p0(ptr nonnull %{{.*}}, i32 8)
+template SYCL_EXTERNAL sycl::vec<unsigned long long, 3> sycl::ext::oneapi::experimental::cuda::ldg(const sycl::vec<unsigned long long, 3> *);
+// CHECK-OPAQUE: tail call <2 x i64> @llvm.nvvm.ldg.global.i.v2i64.p0(ptr %{{.*}}, i32 16)
+// CHECK-OPAQUE: tail call <2 x i64> @llvm.nvvm.ldg.global.i.v2i64.p0(ptr nonnull %{{.*}}, i32 16)
+template SYCL_EXTERNAL sycl::vec<unsigned long, 4> sycl::ext::oneapi::experimental::cuda::ldg(const sycl::vec<unsigned long, 4> *);
+// CHECK-OPAQUE: tail call <2 x i64> @llvm.nvvm.ldg.global.i.v2i64.p0(ptr %{{.*}}, i32 16)
+// CHECK-OPAQUE: tail call <2 x i64> @llvm.nvvm.ldg.global.i.v2i64.p0(ptr nonnull %{{.*}}, i32 16)
+template SYCL_EXTERNAL sycl::vec<unsigned long long, 4> sycl::ext::oneapi::experimental::cuda::ldg(const sycl::vec<unsigned long long, 4> *);
+
+// CHECK-OPAQUE: tail call <4 x i8> @llvm.nvvm.ldg.global.i.v4i8.p0(ptr %{{.*}}, i32 4)
+template SYCL_EXTERNAL sycl::vec<char, 4> sycl::ext::oneapi::experimental::cuda::ldg(const sycl::vec<char, 4> *);
+// CHECK-OPAQUE: tail call <4 x i8> @llvm.nvvm.ldg.global.i.v4i8.p0(ptr %{{.*}}, i32 4)
+template SYCL_EXTERNAL sycl::vec<signed char, 4> sycl::ext::oneapi::experimental::cuda::ldg(const sycl::vec<signed char, 4> *);
+// CHECK-OPAQUE: tail call <4 x i16> @llvm.nvvm.ldg.global.i.v4i16.p0(ptr %{{.*}}, i32 8)
+template SYCL_EXTERNAL sycl::vec<short, 4> sycl::ext::oneapi::experimental::cuda::ldg(const sycl::vec<short, 4> *);
+// CHECK-OPAQUE: tail call <4 x i32> @llvm.nvvm.ldg.global.i.v4i32.p0(ptr %{{.*}}, i32 16)
+template SYCL_EXTERNAL sycl::vec<int, 4> sycl::ext::oneapi::experimental::cuda::ldg(const sycl::vec<int, 4> *);
+
+// CHECK-OPAQUE: tail call <4 x i8> @llvm.nvvm.ldg.global.i.v4i8.p0(ptr %{{.*}}, i32 4)
+template SYCL_EXTERNAL sycl::vec<unsigned char, 4> sycl::ext::oneapi::experimental::cuda::ldg(const sycl::vec<unsigned char, 4> *);
+// CHECK-OPAQUE: tail call <4 x i16> @llvm.nvvm.ldg.global.i.v4i16.p0(ptr %{{.*}}, i32 8)
+template SYCL_EXTERNAL sycl::vec<unsigned short, 4> sycl::ext::oneapi::experimental::cuda::ldg(const sycl::vec<unsigned short, 4> *);
+// CHECK-OPAQUE: tail call <4 x i32> @llvm.nvvm.ldg.global.i.v4i32.p0(ptr %{{.*}}, i32 16)
+template SYCL_EXTERNAL sycl::vec<unsigned int, 4> sycl::ext::oneapi::experimental::cuda::ldg(const sycl::vec<unsigned int, 4> *);
```

Comment:
missing eof here

Reply:
Yeah just a new line.

Reply:
I see! I did just run the the formatter and it changed as such. Is there anything else I should change?

Thanks for reviewing by the way!

Reply:
No problem. Yeah so long as the tests pass, this is fine now.